modules/porous_flow/doc/content/modules/porous_flow/multiapp_fracture_flow_introduction.md

```diff
@@ -0,0 +1,25 @@
+# Fracture flow using a MultiApp approach: Introduction
+
+PorousFlow can be used to simulate flow through fractured porous media in the case when the fracture network is so complicated it cannot be incorporated into the porous-media mesh.  The fundamental premise is *the fractures can be considered as lower dimensional entities within the higher-dimensional porous media*.  This has two consequences.
+
+- Meshing is often simplified and computational speed increased.
+- The fractures do not provide a barrier to flow in their normal direction.  In fact, flows (of both mass and heat) in the normal direction do not even see the fracture as they move through the porous media.  This is certainly true if the fractures have very small width (as befits a lower-dimensional entity) and flow through the fracture is substantially faster than the flow through the porous media.
+
+A [related page](/porous_flow/nomultiapp_flow_through_fractured_media.md) describes how to simulate porous flow in fractured porous media, assuming that the fractures can be incorporated directly into the mesh as lower dimensional elements, for instance, as 2D "fracture" elements inside a 3D "matrix" mesh.  Unfortunately, realistic fracture networks have such complicated geometry that meshing them is difficult, while incorporating their mesh directly into a higher-dimensional mesh is almost impossible.
+Example files are in [multiapp_fracture_flow](https://github.com/idaholab/moose/blob/master/modules/porous_flow/examples/multiapp_fracture_flow).  There is also some [non-PorousFlow documentation](https://github.com/idaholab/moose/blob/master/modules/misc/doc/fracture_flow).
```

Comment:
This weblink is not available.

Reply:
Yea, it won't be there till this is merged.  Do you have a better method for linking?

Reply:
I told him to add it, I wasn't sure what was supposed to work on the moosebuild website.  Is that the same issue the non_pourousFlow documentation link has?

Reply:
Yea, i want to refer to the entire directory, instead of just a particular file.  So i refer to the moose-master blob, which obviously doesn't exist until this gets merged to moose-master.